src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
     />
```

Comment:
You are linking to an SCSS file in your HTML. Browsers can't interpret SCSS directly, you need to compile it to CSS first and then link the CSS file here.